### ISSUE TITLE
Treatment: can we use an enum for the sites and application method?

### ISSUE THREAD

**Issue by Traksewt** [integritysystemscompany/animal_schema#25]
An enum will be useful for interoperability regarding using preset lists and internationalisation/translation. 

There should be medical standards for animal sites. Likewise, the application methods should be a known set.

**Comment by cookeac** [comment]
I agree there should be. I have looked and found plenty of lists but no consistent coding system. Most current farm and vet software I've seen seems to have text fields. Would welcome more discussion and thoughts on this.
